renderer/views/player.js

```diff
@@ -10,6 +10,11 @@ var Bitfield = require('bitfield')
 var TorrentSummary = require('../lib/torrent-summary')
 var {dispatch, dispatcher} = require('../lib/dispatcher')
 
+// Handles volume change by wheel
+var handleVolumeWheel = function (e) {
+  dispatch('changeVolume', (-e.deltaY | e.deltaX) / 500)
+}
```

Comment:
Please use the function declaration style: `function handleVolumeWheel() {}` There is no need to change this function.